src/features/projects/components/ProjectsMap.tsx

```diff
@@ -161,6 +161,17 @@ export default function ProjectsMap(): ReactElement {
     }
   }, [zoomLevel]);
 
+  React.useEffect(() => {
+    if (showProjectList === 'false') {
+      const newViewport = {
+        ...viewport,
+        latitude: 36.96,
+        longitude: 0,
+      };
+      setViewPort(newViewport);
+    }
+  }, [showProjectList, router]);
+
   return (
```

Comment:
@prachigarg19 
1. Not clear why `router` is a dependency here.
2. Please check the combination of `embed` and `showProjectList` as `showProjectList` is not supposed to affect anything without `embed = true`

Reply:
` router` works for initial render.